### PR TITLE
test: add tear_down function, clean up database after test

### PR DESCRIPTION
**Pull request by AdamEECS**

### CODE REVIEW

File: tests/test_pandas.py [hunk 1]
```diff
@@ -50,6 +51,14 @@ def test_pandas_read_from_sqlalchemy_taos():
...
+    conn = taos.connect()
+    db_name = "test"
+    tear_down_database(conn, db_name)
+    conn.close()
+
+
 if __name__ == '__main__':
     test_insert_test_data()
     test_pandas_read_from_rest_connection()
+    test_after()
```
Comment (sunpe) [inline tests/test_pandas.py:64]: pytest does not invoke `main`

File: tests/test_rest_client.py [hunk 2]
```diff
@@ -76,5 +77,14 @@ def test_use_str_timestamp():
...
+    url = os.environ["TDENGINE_URL"]
+    c = RestClient(url)
+    c.sql("drop database if exists test")
+
+
 if __name__ == "__main__":
-    test_show_database()
\ No newline at end of file
+    test_show_database()
+    test_after()
```
Comment (sunpe) [inline tests/test_rest_client.py:90]: pytest does not invoke `main`

File: tests/test_rest_connection.py [hunk 3]
```diff
@@ -127,4 +128,12 @@ def test_tzinfo_timezone():
     c = taosrest.connect(url=url, timezone=tz)
     r = c.query("select * from test.tb")
     for row in r:
-        print(row)  # [datetime.datetime(2022, 7, 26, 13, 56, 58, 746000, tzinfo=datetime.timezone(datetime.timedelta(seconds=28800), 'CST')), -100, -200.3]
+        print(
+            row)  # [datetime.datetime(2022, 7, 26, 13, 56, 58, 746000, tzinfo=datetime.timezone(datetime.timedelta(seconds=28800), 'CST')), -100, -200.3]
+
+
+@check_env
+def test_after():
```
Comment (sunpe) [inline tests/test_rest_connection.py:136]: unused function

**Review by sangshuduo — APPROVED** [review]
LGTM

File: tests/test_pandas.py (hunk 1 above)
Comment (AdamEECS) [reply tests/test_pandas.py]: This line has been removed. Please check the latest version.

File: tests/test_rest_connection.py (hunk 3 above)
Comment (AdamEECS) [reply tests/test_rest_connection.py]: Reformed test_after() with teardown_module(module).

**Review by sunpe — APPROVED** [review]
LGTM

**Pull request merged**